sys/kern/imgact_elf.c

```diff
@@ -525,9 +525,61 @@ __elfN(check_header)(const Elf_Ehdr *hdr)
 	return (0);
 }
 
+#if __has_feature(capabilities)
+/*
+ * Create a reservation for the image activator to map the executable
+ * image into.
+ */
+static int
+__elfN(build_imgact_capability)(struct image_params *imgp,
+    void * __capability *imgact_cap, const Elf_Ehdr *hdr, const Elf_Phdr *phdr,
+    u_long rbase)
+{
+	u_long perm = CHERI_PERM_STORE | CHERI_PERM_GLOBAL |
```

Comment:
If we merge this PR first, I can rebase my #886 changes to include the rounding here.